### PR TITLE
Add conventions for game content capitalization in documentation

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

**Comment by liach** [comment]
So, should `minecraft` be capitalized (or even italicized) or not?

**Comment by i509VCB** [comment]
My documentation in yarn relating to Minecraft by name is capitalized. Given it is a trademark and the name of the game.

File: CONVENTIONS.md [hunk 1]
```diff
@@ -122,6 +122,18 @@ Write quick descriptions for parameter javadocs as well as `@return` tags, with
 
 Use `@link`, `@linkplain` and `@see` tags to refer to other parts of the code.
 
+### Game Content Capitalization
```
Comment (Juuxel) [inline CONVENTIONS.md]:
```suggestion
### Game content capitalization
```

Comment (haykam821) [reply CONVENTIONS.md]: Fixed in d68ecf1db82b231fc59473fc8625d9ad03b811ba

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Comment by haykam821** [comment]
It looks like this was merged into the wrong branch (21w14a instead of 21w15a).

**Comment by Player3324** [comment]
thanks, I'll cherry pick it shortly

**Pull request merged**